### DISCUSSION TITLE
Does LightGCN support to include the node features?

### DISCUSSION THREAD

**Discussion by TheAllUnknown** [pyg-team/pytorch_geometric#8656] [Q&A] [answered]
It seems to me that LightGCN creates node embeddings without considering the node features

**Comment by rusty1s** [comment, marked as answer]
Yes, that's correct. This is how LightGCN is designed. It shouldn't be too hard though to add node features to it, e.g., via concatenation with embeddings.

**Reply by TheAllUnknown** [reply]
I only have node features on the items, does it make sense if I introduce a Linear layer to convert item node features to the same dimension as the user node embedding, and concat them together for lightgcn?

**Reply by rusty1s** [reply]
Yes.
